### DISCUSSION TITLE
Using Incline with zen-mode.nvim

### DISCUSSION THREAD

**Discussion by b0o** [b0o/incline.nvim#77] [Showcase]
I wanted to show Incline while using [zen-mode.nvim](https://github.com/folke/zen-mode.nvim). Here's how I did it:

```lua
require('incline').setup {
  -- ... rest of my config ...
  ignore = {
    floating_wins = false,
    wintypes = function(winid, wintype)
      local zen_view = package.loaded['zen-mode.view']
      if zen_view and zen_view.is_open() then
        return winid ~= zen_view.win
      end
      return wintype ~= ''
    end,
  },
}
```

Now, when zen-mode is active, only the Incline statusline for the floating Zen window will be displayed.

![2024-10-27_15-49-38_region](https://github.com/user-attachments/assets/62430df0-636d-4609-8222-0ddc55e549ec)

Alternatively, if you want to completely hide Incline while zen-mode is active, you can adjust your zen-mode config:
```lua
require('zen-mode').setup {
  on_open = function()
    require('incline').disable()
  end,
  on_close = function()
    require('incline').enable()
  end,
}
```

**Comment by skimask9** [comment]
Would it be possible to share this bubble configuration like in examples?

**Reply by b0o** [reply]
You can find it in my dotfiles, but it's probably not going to be easy to copy 1-for-1 because it depends on several other parts of my config: https://github.com/b0o/nvim-conf/blob/main/lua/user/plugins/incline.lua
